test/integration/init_int_test.go

```diff
@@ -19,54 +23,109 @@ type InitIntegrationTestSuite struct {
 
 func (suite *InitIntegrationTestSuite) TestInit() {
 	suite.OnlyRunForTags(tagsuite.Init, tagsuite.Critical)
-	suite.runInitTest(false, sampleYAMLPython3, "python3")
+	suite.runInitTest(false, "python3", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_SkeletonEditor() {
+func (suite *InitIntegrationTestSuite) TestInit_Path() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(false, sampleYAMLEditor, "python3", "--skeleton", "editor")
+	suite.runInitTest(true, "python3", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_Path() {
+func (suite *InitIntegrationTestSuite) TestInit_Version() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(true, sampleYAMLPython3, "python3")
+	suite.runInitTest(false, "python3@1.0", "python3")
 }
 
-func (suite *InitIntegrationTestSuite) TestInit_Version() {
+func (suite *InitIntegrationTestSuite) TestInit_DisambiguatePython() {
 	suite.OnlyRunForTags(tagsuite.Init)
-	suite.runInitTest(false, sampleYAMLPython3, "python3@1.0")
+	suite.runInitTest(true, "python", "python3")
+	suite.runInitTest(true, "python@3.10.0", "python3")
+	suite.runInitTest(true, "python@2.7.18", "python2")
 }
 
-func (suite *InitIntegrationTestSuite) runInitTest(addPath bool, config string, language string, args ...string) {
+func (suite *InitIntegrationTestSuite) runInitTest(addPath bool, lang string, expectedConfigLanguage string, args ...string) {
```

Comment:
With all the projects this is now creating on the platform we should call `ts.DeleteUUIDProjects()` as a cleanup. eg. https://github.com/ActiveState/cli/blob/d35115ca6fd927436a27ccf27d53db87758c1865/test/integration/delete_uuid_projects_int_test.go#L19-L19

It seems vital that this is called before `ts.Close()`.

Reply:
That is happening behind the scenes here:
https://github.com/ActiveState/cli/pull/2483/files#diff-43151b4055cc3065eb74749285d1a52636e027a42052a542e98c1992950a6bf2R63

Reply:
Whoops, missed that!